types/hapi/test/route/additional-options.ts

```diff
@@ -6,6 +6,29 @@ var authConfig: Hapi.RouteAdditionalConfigurationOptions = {
   app: {}
 };
 
+var extConfigSingle: Hapi.RouteAdditionalConfigurationOptions = {
+  ext: {
+    type: 'onPreAuth',
+    method: <Hapi.ServerExtRequestHandler> function (request, reply) {
+      reply('ok');
+    }
+  }
+}
+
+var extConfigMulti: Hapi.RouteAdditionalConfigurationOptions = {
+  ext: [{
+    type: 'onPreAuth',
+    method: <Hapi.ServerExtRequestHandler> function (request, reply) {
+      reply('ok');
+    }
+  }, {
+    type: 'onPostAuth',
+    method: <Hapi.ServerExtRequestHandler> function (request, reply) {
+      reply('ok');
+    }
+  }]
```

Comment:
Hu, somehow vscode autodetected that wrong, I wonder why the repo doesn't include rules for `.ts` files in the editorconfig file.